src/pokemon.c

```diff
@@ -2158,3 +2159,138 @@ u32 MonTryLearnMoveOnLevelUp(struct PartyPokemon *mon, int * last_i, u16 * sp0)
     sys_FreeMemoryEz(levelUpLearnset);
     return ret;
 }
+
+const u8 sTrainerGenders[] = {
+    TRAINER_MALE,   // TRAINERCLASS_PKMN_TRAINER_ETHAN
+    TRAINER_FEMALE, // TRAINERCLASS_PKMN_TRAINER_LYRA
+    TRAINER_MALE,   // TRAINERCLASS_YOUNGSTER
```

Comment:
can do like
```c
[TRAINERCLASS_PKMN_TRAINER_ETHAN] = TRAINER_MALE,
[TRAINERCLASS_PKMN_TRAINER_LYRA] = TRAINER_FEMALE,
[TRAINERCLASS_YOUNGSTER] = TRAINER_MALE,
```
to prevent misalignment

Reply:
Just updated the PR to do this : )